### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by maujin111**
add issue templates

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
+ - Version [e.g. 22]
```
Comment (Zughy) [inline .github/ISSUE_TEMPLATE/bug_report.md:29]: I'd say
```
**OS/Hardware**
- Operative System:
- CPU:
- GPU:
```

and I'd move the LibreSprite version at the very beginning (see the other comment)

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
```
Comment (Zughy) [inline .github/ISSUE_TEMPLATE/bug_report.md:9]:
```
**LibreSprite version**
On the bottom left corner when you open the software
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
```
Comment (Zughy) [inline .github/ISSUE_TEMPLATE/bug_report.md:24]: I'd remove this section, mentioning screenshots and videos in **Additional context**

**Comment by Zughy** [comment]
Uhm, I see my suggestions resolved but no new commits nor comments

**Comment by maujin111** [comment]
> Uhm, I see my suggestions resolved but no new commits nor comments

Done

**Comment by Zughy** [comment]
I'd approve it but... I don't have the power to :\

**Pull request closed without merging**